### ISSUE TITLE
Broken Support for Multimedia Fusion programs (?)

### ISSUE THREAD

**Issue by Chopppers** [LAK132/SourceExplorer#95]
**Describe the bug**
Source Explorer "crashes" during the middle of a dump and cannot be recovered. when checking the folder dump, it seemed to have stopped on "sounds", but fully exported all "sprites" before hand.

**To Reproduce**
1. Drag and drop a game that uses multimedia fusion (description should be "Multimedia Fusion Stand Alone Application")
2. Start dump

**Expected behavior**
should have exported all assets besides sprites

**Additional context**
was attempting to extract "Sonic: Time Attacked", a game released on September 9th 2003.

[link to crash log](https://file.io/nrjQ5Cy6UdJV)

**Comment by Chopppers** [comment]
small update: it seems that it exclusively can not dump sound, but from the looks of it, an older version of source explorer can?? didn't realize there was also a similar issue that someone else posted

**Comment by MrKsoft** [comment]
Yeah, this seems to be related to my issue (#81 ).  There have been improvements in the latest code commits that allow the apps I have tested to extract images again but it looks like maybe the work is not yet complete.

I do recall that in the old versions, there were still some issues with extracting sounds anyway - namely some extracted games would produce unplayable WAV files.  Looked like something with them being compressed to ADPCM and the decompressed header being wrong, but I couldn't pinpoint the exact issue and didn't want to submit a bug report against such an old version of Source Explorer.
